src/common/cd-gender-component.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+<div>
+  <select class="select-box form-control" data-vv-name="gender" data-vv-validate-on="blur" v-validate="'required'" v-model="genderSelect">
+    <option value="" selected data-default disabled>{{ $t('Select an option') }}</option>
+    <option value="male">{{ $t('Male') }}</option>
+    <option value="female">{{ $t('Female') }}</option>
+    <option value="prefer not to answer">{{ $t('Prefer not to answer') }}</option>
+    <option value="specify">{{ $t('Specify Identity') }}</option>
+  </select>
+  <input v-show="specifyGender" class="form-control" v-model="genderInput" :placeholder="$t('Identify as...')" data-vv-name="gender" data-vv-validate-on="blur" v-validate="specifyGender ? 'required' : ''"/>
+<p class="gender-err text-danger" v-show="errors.has('gender:required')">{{ $t('Gender is required') }}<br/><a v-on:click="showWhy">{{ $t('Why is this required? Click here to find out more') }}</a></p>
```

Comment:
Where is the indentation gone? :fox_face:

Reply:
woops, will fix that